frontend/packages/knative-plugin/src/components/add/import-types.ts

```diff
@@ -1,9 +1,8 @@
 export enum EventSources {
   CronJobSource = 'CronJobSource',
-  ContainerSource = 'ContainerSource',
   ApiServerSource = 'ApiServerSource',
   KafkaSource = 'KafkaSource',
-  CamelkSource = 'CamelSource',
+  CamelSource = 'CamelSource',
```

Comment:
You can just remove this line from the enum and you have your list of event sources that have their own form sections.

Reply:
done